src/odemis/gui/cont/streams.py

```diff
@@ -160,6 +164,9 @@ def __init__(self, stream_bar, stream, tab_data_model, show_panel=True, view=Non
         elif hasattr(stream, "emission"):  # only emission
             self._add_emission_ctrl()
 
+        if len(self.entries) > num_settings_entries:  # TODO: also only if other controls after
```

Comment:
If TODO's are not done in the near future a more elaborate description would be nice so other people also know the details

Reply:
This refers to the TODO which is just above, line 152. I think that comment is sufficiently clear that someone would understand (at least, if seeing the GUI at the same time).